modules/light-clients/06-solomachine/internal/keeper/keeper.go

```diff
@@ -1,38 +1,23 @@
 package keeper
 
 import (
-	"fmt"
-
-	"cosmossdk.io/store/prefix"
-	storetypes "cosmossdk.io/store/types"
-
 	"github.com/cosmos/cosmos-sdk/codec"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-
-	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
 )
 
 // Keeper defines the 06-solomachine Keeper.
+// TODO(damian): delete the keeper and put the codec on the LightClientModule?
```

Comment:
+1 keeper may only be necessary if a msg server is added (which for solo machine I'm not sure we need)